src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html

```diff
@@ -243,6 +243,21 @@ <h5 jhiTranslate="artemisApp.exercise.export.options"></h5>
                             </ng-template>
                         </ngx-datatable-column>
                     }
+                    <ngx-datatable-column prop="" [sortable]="true" [minWidth]="100" [width]="100">
+                        <ng-template ngx-datatable-header-template>
+                            <span class="datatable-header-cell-wrapper d-inline-block w-100 text-center" (click)="controls.onSort('results?.[0]?.assessmentNote?.note')">
+                                <fa-icon [icon]="faComment" [ngbTooltip]="'artemisApp.assessment.assessmentNote' | artemisTranslate" />
+                                <fa-icon [icon]="controls.iconForSortPropField('results?.[0]?.assessmentNote?.note')" />
+                            </span>
+                        </ng-template>
+                        <ng-template ngx-datatable-cell-template let-value="value">
+                            @if (value.results?.[0]?.assessmentNote?.note) {
```

Comment:
The other columns here use the last result instead of the first one. Is that intended?

Reply:
I don't think so, but I do not understand how there can be more than one result here. How can I get such a situation?

Reply:
`value` is a participation and they are retrieved with this method: `findByExerciseIdWithLatestAndManualResults`
As you can see all non automatic results are retrieved, which can be the case for complaints or multiple assessment rounds during an exam

Reply:
Thank you, with complaints I was able to trigger the bug, where you see the note here in the list, but not in the details view :sweat_smile: . I will switch to `last()` to fix that.

Reply:
Fixed with https://github.com/ls1intum/Artemis/pull/8552/commits/79dab282ede2cd00440f4ecb0a649596e3ea02d2